readme.txt

```diff
@@ -79,6 +81,9 @@ The source code is freely available [in GitHub](https://github.com/Automattic/wo
 
 == Changelog ==
 
+= 2.7.1 - 2024-xx-xx =
+* Update - Update readme.txt file to indicate that a new Woo Shipping plugin is available for download.
+
```

Comment:
Same as the changelog file, no need to add this.

Reply:
Removed in ab3dacaf